lib/libzfs/libzfs_pool.c

```diff
@@ -4215,11 +4209,11 @@ zpool_get_errlog(zpool_handle_t *zhp, nvlist_t **nverrlistp)
 		nvlist_free(nv);
 	}
 
-	free((void *)(uintptr_t)zc.zc_nvlist_dst);
+	free(buf);
 	return (0);
 
 nomem:
-	free((void *)(uintptr_t)zc.zc_nvlist_dst);
+	free(buf);
```

Comment:
Hm, maybe I'm missing it... but I don't see where the `nomem:` label is used in the function.

Reply:
see lines 4194-4207, which were not changed and are hidden in the github UI